draft-irtf-cfrg-vdaf.md

```diff
@@ -4515,19 +4526,36 @@ differential privacy.
 
 ## Requirements for XOFs {#xof-vs-ro}
 
-XofShake128 is designed to be indifferentiable from a random oracle {{MRH04}},
-making it a suitable choice for most situations, in particular wherever the XOF
-is modeled as a random oracle in existing security analysis.
-
-XofFixedKeyAes128 is known to be differentiable from a random oracle
-{{GKWWY20}} and is therefore NOT RECOMMENDED for general use. It is used only by
-IdpfPoplar ({{idpf-poplar}}) whose security analysis does not require a random
-oracle.
-
-> TODO(issue #216) Justify this claim.
-
-> OPEN ISSUE: We may want to drop the common interface for XOFs and random
-> oracles. See issue #159.
+As described in {{xof}}, our constructions rely on an interface for eXtendable
+Output Functions (XOFs). In the security analyses of our protocols, these are
+mostly modelled as random oracles. XofShake128 is designed to be
+indifferentiable from a random oracle {{MRH04}}, making it a suitable choice
+for most situations.
+
+The one exception is the Idpf implementation IdpfPoplar {{idpf-poplar}}.
+Here, a random oracle is not needed to prove privacy, since the analysis of
+{{BBCGGI21}}, Proposition 1, only requires a Pseudorandom Generator (PRG).
+As observed in {{GKWY20}}, a PRG can be instantiated from a correlation-robust
+hash function `H`. Informally, correlation robustness requires that for a random
+`r`, `H(xor(r, x))` is indistinguishable from a random function for any `x`.
+A PRG can therefore be constructed as
+`PRG(r) = H(xor(r, 1)) || H(xor(r, 2)) || ...`,
```

Comment:
Do you intend for this to be on its own line? If so, then:
```suggestion

~~~
PRG(r) = H(xor(r, 1)) || H(xor(r, 2)) || ...`
~~~

```
If you want this math to be in-lined, then, make sure lines break at 80 characters.